rewards-configuration.default.yml

```diff
@@ -27,29 +27,39 @@ formattingEvaluator:
     td: 1
     hr: 0
   multipliers:
-    - type: [ISSUE, ISSUER]:
+    - type: [ISSUE, ISSUER, TASK]:
+      formattingMultiplier: 1
+      wordValue: 0.1
+    - type: [ISSUE, ISSUER, COMMENTED]:
       formattingMultiplier: 1
       wordValue: 0.2
-    - type: [ISSUE, ASSIGNEE]:
+    - type: [ISSUE, ASSIGNEE, COMMENTED]:
       formattingMultiplier: 0
       wordValue: 0
-    - type: [ISSUE, COLLABORATOR]:
+    - type: [ISSUE, COLLABORATOR, COMMENTED]:
```

Comment:
Perhaps it will make the config more expressive if you add other properties?

For example 

```yml

targets:
   views:
      - ISSUE
      - PULL
   actors:
      - ISSUER 
      - COLLABORATOR 
   actions: 
      - COMMENTED

```

Reply:
`observe` would help to chose what we want to include and ignore? We surely can add more properties as it goes. The purpose of this section is to attribute different multipliers based on the type of comment, mimicking the old [codebase](https://github.com/ubiquibot/comment-incentives/blob/525fcdccc8f0d9032a41cdaed3b4fbc6514d44b6/src/handlers/issue/comment-scoring-by-contribution-style.ts)

Reply:
I am aware of its purpose. I am proposing to make the config more expressive so that its more intuitive to work with.